OWML.ModLoader/ModLoader.cs

```diff
@@ -34,7 +35,8 @@ public static void LoadMods()
             var harmonyHelper = new HarmonyHelper(logger, console);
             var events = new ModEvents(logger, console, harmonyHelper);
             var inputHandler = new ModInputHandler(logger, console, harmonyHelper, owmlConfig, events);
-            var menus = new ModMenus(console, events, inputHandler);
+            var owmlData = new ModData(owmlManifest, null, null);
```

Comment:
This is horrible, but can't think of a nice way to avoid it

Reply:
The mod data object is used by the base config menu which I override for the OWML config menu.
For the OWML config menu, mod data is only used once, in the constructor of the base class, which uses the manifest only.
The ways I've thought of refactoring it would lead to all sorts of side effects.

Reply:
well... It's not like you can't just add '=null' to contstructor itself or create a separate constructor

Reply:
This is an attempt: https://github.com/amazingalek/owml/pull/148